src/layer/marker/Icon.Default.js

```diff
@@ -42,19 +42,25 @@ export var IconDefault = Icon.extend({
 		return (this.options.imagePath || IconDefault.imagePath) + Icon.prototype._getIconUrl.call(this, name);
 	},
 
+	_stripUrl: function (path) {	// separate function to use in tests
```

Comment:
Might be more clear to say `// standalone function so it can be tested independently`